### PR TITLE
CAPI -1116: added exclusiveDeal

### PR DESCRIPTION
**Pull request by smritiprasad**

### CODE REVIEW

File: apis/one-connect/hotel/v1.0/_dataStructure/data-structures.apib [hunk 1]
```diff
@@ -106,10 +106,14 @@
 + supplierHotelId: `1234497` (string) - Indicates the supplier hotel ID that is mapped with the clarifi hotel ID.
 
 ## moreResultsSearchRes
-+ moreResults: `false` (boolean) - Indicates whether all the requested suppliers have returned all the search results. If this field returns `true`, this means that more results are available from some suppliers and another `Get Hotel Search Results` API call can be made to retrieve the updated list of search results. If this field returns `false`, this means that all the search results from all the requested suppliers are already provided in the current result set.
++ moreResults: false (boolean) - Indicates whether all the requested suppliers have returned all the search results. If this field returns `true`, this means that more results are available from some suppliers and another `Get Hotel Search Results` API call can be made to retrieve the updated list of search results. If this field returns `false`, this means that all the search results from all the requested suppliers are already provided in the current result set.
+  + True
```
Comment (rohit77p) [inline apis/one-connect/hotel/v1.0/_dataStructure/data-structures.apib]: Global comment: We do not give list of values for value type boolean. The boolean itself means either true or false.

File: apis/one-connect/hotel/v1.0/_dataStructure/data-structures.apib [hunk 2]
```diff
@@ -1761,7 +1775,7 @@ There are two types of rates:<br><br>- `perBookingRate`<br>- `perRoomRate`<br><b
 + allowHotelIds: (array[string]) - Contains the list of hotel IDs that must be included in the search results. This is the unique ID that clarifi has assigned to the hotel. Hotels that do not match these IDs will not be returned in the response.<br><br>Type: Array of strings
 
 ## allowNotAvailableHotels
-+ allowNotAvailableHotels: `true` (boolean) - If you want to search only for those hotels that have the availability or room rate information available from the supplier, set the `includeHotelsWithoutRates` field to `false`. If you set the `includeHotelsWithoutRates` field to `true`, the search results will include hotels regardless of the availability or room rate information.<br><br>
++ allowNotAvailableHotels: true (boolean) - If you want to search only for those hotels that have the availability or room rate information available from the supplier, set the `includeHotelsWithoutRates` field to `false`. If you set the `includeHotelsWithoutRates` field to `true`, the search results will include hotels regardless of the availability or room rate information.<br><br>
```
Comment (rohit77p) [inline apis/one-connect/hotel/v1.0/_dataStructure/data-structures.apib]: Are we talking about two separate fields `allowNotAvailableHotels` and `includeHotelsWithoutRates`? This seems a bit confusing.

Comment (smritiprasad) [reply apis/one-connect/hotel/v1.0/_dataStructure/data-structures.apib]: Updated.

File: apis/one-connect/hotel/v1.0/_dataStructure/data-structures.apib (hunk 1 above)
Comment (smritiprasad) [reply apis/one-connect/hotel/v1.0/_dataStructure/data-structures.apib]: Updated.

**Pull request merged**